AzurePipelinesToGitHubActionsConverter/AzurePipelinesToGitHubActionsConverter.Core/Conversion/StepsProcessing.cs

```diff
@@ -197,11 +198,11 @@ public GitHubActions.Step ProcessStep(AzurePipelines.Step step)
                         }
                     }
                 }
-                if (string.IsNullOrEmpty(step.continueOnError) == false)
+                if (step.continueOnError)
```

Comment:
so I could have gone further and refactored this yes, but the/ first rule of pull requests is not to re-write the code beyond comprehension... I just wanted to get my changes IN :p

Reply:
We still need to fix this. if ContinueOnError is set to false, it will not set the GitHubStep property to false, only if it's true.

Reply:
The default value for a non-nullable boolean property is false, so the if condition has no effect either way. I left the if condition in as part of the 'change as little as possible to make PR easier' mantra. I have removed the if condition now.

Reply:
ahh. you are right again. If it's true it will be set, if it's false, it will be false. 

Is the side effect that every GitHub action will have this property now?

Reply:
Yes when using YamlDotNet every property will specify as false, however you add a DefaultValue which will then hide that :) Generally to keep the YAML output as readable as possible you should use defaults to hide implicit property settings, although sometimes this can lead to fun errors if you hardcode in the wrong default!

```csharp
[DefaultValue("succeeded()")]
public string condition { get; set; }
[DefaultValue(0)]
public int timeoutInMinutes { get; set; } = 0;
[DefaultValue(1)]
public int cancelTimeoutInMinutes { get; set; } = 1;
```

Reply:
My question was around the idea that now these properties are set to something, instead of nothing. Before when it was nothing, I didn't set the property and it didn't render. I was afraid these would now render as they have a default. They don't seem to be so far, which is good.